src/DataTable/FilterStatus.jsx

```diff
@@ -23,7 +24,15 @@ const FilterStatus = ({
         size={size}
         onClick={() => setAllFilters([])}
       >
-        {clearFiltersText}
+        {clearFiltersText === undefined
```

Comment:
Good question! Initially I tried to check for falsey value but decided against it because there is a specific [test case](https://github.com/openedx/paragon/blob/master/src/DataTable/tests/FilterStatus.test.jsx#L64) which tests that no text gets rendered if you pass an empty string, it was failing.

I could change that test but I thought that _maybe_ there is a use case for not displaying any text..

Reply:
Hmm, interesting. I suppose that _could_ be possible; best to handle it explicitly I guess, considering there is a specific test for it 🤷‍♂️ Though, I cant imagine why you'd want to do this haha.